actions/playerHeroesActions.js

```diff
@@ -1,8 +1,10 @@
 import fetch from 'isomorphic-fetch';
 import { HOST_URL } from './';
 import { playerHeroes } from '../reducers';
+import { getUrl, addQueryString, defaultOptions, getModifiedOptions } from './utility';
 
-const url = (playerId, queryString) => `/api/players/${playerId}/heroes?${queryString}`;
+const excludedOptions = ['limit'];
```

Comment:
yea that makes sense, i'll add it back in. The limiting is just by last played right now, right (since there is no sorting on api)?